modules/log6x-loki-sizing.adoc

```diff
@@ -0,0 +1,88 @@
+// Module is included in the following assemblies:
+// * observability/logging/logging-6.0/
+
+:_mod-docs-content-type: CONCEPT
+[id="log6x-loki-sizing_{context}"]
+= Loki deployment sizing
+
+Sizing for Loki follows the format of `1x.<size>` where the value `1x` is number of instances and `<size>` specifies performance capabilities.
+
+[IMPORTANT]
+====
+It is not possible to change the number `1x` for the deployment size.
+====
+
+.Loki sizing
+[cols="1h,5*",options="header"]
+|===
+|
+|1x.demo
+|1x.pico [6.1+ only]
+|1x.extra-small
+|1x.small
+|1x.medium
+
+|Data transfer
+|Demo use only
+|50GB/day
+|100GB/day
+|500GB/day
+|2TB/day
+
+|Queries per second (QPS)
+|Demo use only
+|1-25 QPS at 200ms
+|1-25 QPS at 200ms
+|25-50 QPS at 200ms
+|25-75 QPS at 200ms
+
+|Replication factor
+|None
+|2
+|2
+|2
+|2
+
+|Total CPU requests
+|None
+|5 vCPUs
+|14 vCPUs
+|34 vCPUs
+|54 vCPUs
+
+|Total CPU requests if using the ruler
+|None
+|5.5 vCPUs
+|16 vCPUs
+|42 vCPUs
+|70 vCPUs
+
+|Total memory requests
+|None
+|20Gi
+|31Gi
+|67Gi
+|139Gi
+
+
+|Total memory requests if using the ruler
+|None
+|21Gi
+|35Gi
+|83Gi
+|171Gi
+
+|Total disk requests
+|40Gi
+|430Gi
```

Comment:
I'm getting 590 GiB without and 910 GiB with the ruler.
I think we forgot to adjust the ingester's PVC sizes when we added the third ingester.

For now, let's go with the sizes that will be in the release, we probably need to adjust this down in a z-stream.

Reply:
@xperimental - it seems a bit off that the disk request for pico is more than the larger sizes, I would expect that to potentially cause confusions - is there there a callout we can add to mitigate?